system-int-test/test/cmr/system_int_test/ingest/service_ingest_test.clj

```diff
@@ -0,0 +1,99 @@
+(ns cmr.system-int-test.ingest.service-ingest-test
+  "CMR service ingest integration tests."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :refer [are3]]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.ingest-util :as ingest-util]
+   [cmr.system-int-test.utils.metadata-db-util :as mdb]
+   [cmr.system-int-test.utils.service-util :as service-util]))
+
+(use-fixtures :each (ingest-util/reset-fixture))
+
+(deftest create-service-ingest-test
```

Comment:
Updates for Variables has it's own ticket; I'm assuming we'll be doing updates for Services in the same way (in its own ticket) ... @chris-durbin can confirm.